k8s/stset/lrp_to_statefulset.go

```diff
@@ -261,6 +260,14 @@ func getContainerResources(cpuWeight uint8, memoryMB, diskMB int64) corev1.Resou
 	}
 }
 
+func NewMebibyteQuantity(miB int64) resource.Quantity {
```

Comment:
On reflection, since we are returning a struct, not a pointer here, this function should probably be named MebibyteQuantity instead of NewMebibyteQuantity